paunchy-llamas/waddle
# Phone & Twilio stuff is working

HOW TO TEST IT:

Right now the Foursquare API isn't set up so it will use the Nathaniel test data.
That test data must be added to your instance of mongo in order for the database query to work.
 Sign up with an account with the username Nathaniel and your own phone number in the phone field, after registering it with Twilio. (If you want to actually get a text message you have to do this. A Twilio paid account does not require that 'to' numbers be registered FYI.)

You must sign up with a different user to get Nathaniel returned (otherwise it will return Sloth, the second test-data user). 

That's all there is to it. 

Make sure to put only numbers into the phone field for any dummy accounts you create. There is no form validation so you will get a 'failed to cast as number' error otherwise. As for the Twilio call, it only calls if the provided number is 10 digits, but that's it for validation.
